### PR TITLE
Fix `svn` deployment by using `wp-svn` orb 

### PR DESCRIPTION
**Pull request by kienstra**
Uses our [wp-svn Orb](https://circleci.com/developer/orbs/orb/studiopress/wp-svn), like [Genesis Connect Woocommerce does](https://github.com/studiopress/genesis-connect-woocommerce/blob/develop/.circleci/config.yml#L40C9-L48)

### How to test

* Look at the `svn stat` [from the dry run](https://app.circleci.com/pipelines/github/studiopress/genesis-simple-sidebars/124/workflows/759e2d4b-4dbd-4e1d-9bc1-8a47594538d3/jobs/278)

### Next steps
After this is merged, Nick publishes a new `2.2.3` release, and we'll see if this deployed to production correctly

### Documentation
No documentation required.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -138,52 +37,12 @@ workflows:
...
-                - master
-      - checks:
-          requires:
-            - checkout
-          filters:
-            branches:
-              only:
-                - master
-      - deploy_svn_branch:
+      - wp-svn/deploy:
```
Comment (kienstra) [inline .circleci/config.yml:35]: This is the main fix. 

It uses [our Orb](https://circleci.com/developer/orbs/orb/studiopress/wp-svn), which handles almost all of the `svn` details.

File: .circleci/config.yml [hunk 2]
```diff
@@ -70,63 +14,18 @@ executors:
...
-  checkout:
-    executor: base
-    steps:
-      - mkdir_artifacts
-      - checkout:
-          path: src
-      - persist_to_workspace:
-          root: /tmp
-          paths:
-            - src
```
Comment (kienstra) [inline .circleci/config.yml:82]: A separate `checkout` job isn't needed, `checkout` can be a step in each job

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,63 +1,7 @@
...
-    description: "Create a SVN tag"
-    steps:
-      - set_verision_variable
-      - run: svn cp trunk tags/${VERSION}
-
-  svn_commit:
-    description: "Commit changes to SVN"
-    steps:
-      - set_verision_variable
-      - run: svn ci -m "Tagging ${VERSION} from Github" --no-auth-cache --non-interactive --username "${SVN_USERNAME}" --password "${SVN_PASSWORD}"
```
Comment (kienstra) [inline .circleci/config.yml:60]: Our `wp-svn` Orb handles this logic

**Review by dreamwhisper — APPROVED** [review]

**Comment by kienstra** [comment]
Thanks, Jen!

**Review by nickcernis — APPROVED** [review]

**Pull request merged**